templates/template-test.js

```diff
@@ -13,6 +13,11 @@ if (typeof spec.mark === 'string') {
 	spec.mark = { type: spec.mark };
 }
 
+// Change spec.mark from a string to an object, which is what vega-lite-api creates
+if (typeof spec.spec === 'object' && typeof spec.spec?.mark === 'string') {
```

Comment:
Could you add this to the `Notes` section of the readme template?

Reply:
Pushed new updates on readme template